test/functional/api/cas/cache.py

```diff
@@ -88,10 +89,29 @@ def get_flush_parameters_acp(self):
 
     # Casadm methods:
 
-    def get_cache_statistics(self,
-                             io_class_id: int = None,
-                             stat_filter: List[StatsFilter] = None,
-                             percentage_val: bool = False):
+    def get_part_statistics(self,
+                            io_class_id: int,
+                            stat_filter: List[StatsFilter] = None,
+                            percentage_val: bool = False):
+        stats = get_statistics(self.cache_id, None, io_class_id,
+                               stat_filter, percentage_val)
+        return IoClassStats(stats)
+
+    def get_statistics(self,
+                       io_class_id: int = None,
```

Comment:
Since you have separate method for retrieving ioclass stats, this method shouldn't have `io_class_id` param.